### PR TITLE
added bbp-morphology-workflow

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: bbp/nse/bbp-morphology-workflow/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  config,
```
Comment (tristan0x) [inline bbp/nse/bbp-morphology-workflow/default.nix]: To be pedantic, I would sort this section.

File: bbp/nse/bbp-morphology-workflow/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    name = "${pname}-${version}";
+
+    src = fetchgitPrivate {
+        url = config.bbp_git_ssh + "/platform/bbp-morphology-workflow";
+        rev = "4808570b7914dc621db45a2709f62cfcea73d57c";
+        sha256 = "0adll1vz0nb1y9rrmv8w8r8h66pb9nb4id1vidpxs0aa9j19cba5";
+    };
+
+    checkInputs = with pythonPackages; [
+        pylint
```
Comment (tristan0x) [inline bbp/nse/bbp-morphology-workflow/default.nix]: I would sort this section

File: bbp/nse/bbp-morphology-workflow/default.nix [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        coverage
+        pycodestyle
+        mock
+        nose
+    ];
+
+    propagatedBuildInputs = with pythonPackages; [
+        neurom
```
Comment (tristan0x) [inline bbp/nse/bbp-morphology-workflow/default.nix]: I would sort this section

File: bbp/nse/bbp-morphology-workflow/default.nix [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        neurodamus
+        bbpsdk-legacy
+        muk
+        bluerepairsdk
+        morphscale
+        bluejittersdk
+    ];
+
+    checkPhase = ''
+        nosetests morphology_repair_workflow/tests
```
Comment (tristan0x) [inline bbp/nse/bbp-morphology-workflow/default.nix]: You should keep the pre/post hook:

```
checkPhase = ''
  runHook preCheck
  nosetests morphology_repair_workflow/tests
  runHook postCheck
'';

**Comment by tristan0x** [comment]
retest this please
build package: foo
(this is just a test)

File: bbp/nse/bbp-morphology-workflow/default.nix (hunk 4 above)
Comment (mgeplf) [reply bbp/nse/bbp-morphology-workflow/default.nix]: what's that for?

Maybe it would be worthwhile to add tips/information like this to a nix page?

Comment (tristan0x) [reply bbp/nse/bbp-morphology-workflow/default.nix]: Here is the Nix documentation about the [check phase](https://nixos.org/nixpkgs/manual/#ssec-check-phase)
`preCheck` and `postCheck` could be defined elsewhere.

**Pull request merged**